### ISSUE TITLE
Block inserter not appearing above Classic Block

### ISSUE THREAD

**Issue by maddisondesigns** [WordPress/gutenberg#7917]
**Describe the bug**
After inserting a Classic Block I can't insert a block above it as the Block inserter doesn't appear.

![gutenberg_cantinsertblockbeforeclassicblock](https://user-images.githubusercontent.com/2078245/42613552-e1ecb36a-85e4-11e8-8dee-2e2a938f4b77.gif)

**To Reproduce**
1. Create new page
2. Insert paragraph block with content
3. Insert Classic Block with content
4. Try to insert Block above Classic Block

**Expected behavior**
Block inserter should appear above all blocks

_macOS Sierra 10.12.1
Firefox Quantum 61.0.1 (64-bit)
Gutenberg 3.2.0_

**Comment by designsimply** [comment]
Tested and confirmed that I cannot see any block inserter for the classic block.

**Video:** [42s](https://cloudup.com/c0bd0xD8SJi)
<sup>Seen at http://alittletestblog.com/wp-admin/post.php?post=13963&action=edit running WordPress 4.9.7 and Gutenberg 3.2.0 using Firefox 61.0.1 on macOS 10.13.5.</sup>

**Comment by maddisondesigns** [comment]
They also won't randomly show up on other blocks as well. Just had an issue where the block inserter wouldn't appear on a Paragraph block that was sitting between an image and another Paragraph block.

**Comment by SuperGeniusZeb** [comment]
I think this is a duplicate of #7884.

**Comment by maddisondesigns** [comment]
@SuperGeniusZeb Nope, because the inserter doesn't appear on Blocks with Toolbars either. It's not just an issue on Blocks without Toolbars

**Comment by SuperGeniusZeb** [comment]
@maddisondesigns Is that not exactly what #7884 is about? The sibling inserter currently never appears above the currently selected block (because it was assumed that the toolbar would usually get in the way, except that some blocks do not have toolbars or in the case of the Classic block do not have the same kind of toolbar).

**Comment by maddisondesigns** [comment]
@SuperGeniusZeb All #7884 seems to talk about is that it it doesn't appear on blocks without toolbars. That's not the case. It also doesn't appear on Classic Blocks (which does have a toolbar) and also (as mentioned in one of my other comments), it sometimes doesn't appear on other blocks as well, such as the Paragraph Block. They seem like two different issues to me.

**Comment by designsimply** [comment]
I tested again and found that the inserter _does_ appear above classic blocks if they are not selected:

![screen shot 2018-07-17 at tue jul 17 7 31 16 pm](https://user-images.githubusercontent.com/1119271/42854256-41cf2f58-89f8-11e8-988d-f617b590c2e6.png)
<sup>Seen at http://alittletestblog.com/wp-admin/post.php?post=13999&action=edit running WordPress 4.9.7 and Gutenberg 3.2.0 using Firefox 61.0.1 on macOS 10.13.5.</sup>

Closing as a duplicate because the inserter does sometimes appear for the classic block, because this issue now has a bit of extra noise in it 🙂 compared to the other issue, and because I think @SuperGeniusZeb is correct that the fix for #7884 will likely address the issue raised here. If it doesn't, we can re-open this one after #7884 is fixed!
